src/clp_ffi_js/ir/StructuredIrStreamReader.cpp

```diff
@@ -132,7 +133,8 @@ auto StructuredIrStreamReader::decode_range(size_t begin_idx, size_t end_idx, bo
             );
             json_str = std::string(cEmptyJsonStr);
         } else {
-            json_str = json_result.value().dump();
+            json_str = json_result.value()
```

Comment:
The magic values seem a bit odd. Shall we consider creating a helper for this? e.g.,

Add this to the anonymous namespace:
```
constexpr auto cDumpWithReplace = [](auto const& json_obj) {
    return json_obj.dump(-1, ' ', false, nlohmann::json::error_handler_t::replace);
};
```